contracts/multicaller/Multicall3.sol

```diff
@@ -0,0 +1,216 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Please add either inside this file or as a adjoining _readme_ file the path to the github source including commit hash